geomesa-tools/bin/geomesa-kafka

```diff
@@ -0,0 +1,173 @@
+GEOMESA_OPTS="-Duser.timezone=UTC -DEPSG-HSQL.directory=/tmp/$(whoami)"
+GEOMESA_DEBUG_OPTS="-Xmx8192m -XX:MaxPermSize=512m -XX:-UseGCOverheadLimit -Xdebug -Xnoagent -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=9898"
+GEOMESA_CP=""
+JAVA_LIBRARY_PATH=""
+
+# Define GEOMESA_HOME and update the PATH if necessary.
+if [ -z "$GEOMESA_HOME" ]; then
+setGeoHome
+else
+    echo >&2 "Using GEOMESA_HOME = $GEOMESA_HOME"
+    if [[ $1 = configure ]]; then
+        read -p "Do you want to reset this? Y\n " -n 1 -r
+        if [[  $REPLY =~ ^[Yy]$ ]]; then
+            echo >&2 ""
+            setGeoHome
+        fi
+        echo >&2 ""
+    fi
+fi
+
+# GEOMESA paths, GEOMESA_LIB should live inside GEOMESA_HOME, but can be pointed elsewhere
+if [ -z "$GEOMESA_LIB" ]; then
+    GEOMESA_LIB=${GEOMESA_HOME}/lib
+elif [[ $1 = configure ]]; then
+    message="Warning: GEOMESA_LIB was already set, probably by a prior configuration."
+    message="${message}\n The current value is ${GEOMESA_LIB}."
+    echo >&2 ""
+    echo -e >&2 "$message"
+    echo >&2 ""
+    read -p "Do you want to reset this to ${GEOMESA_HOME}/lib? Y\n " -n 1 -r
+    if [[  $REPLY =~ ^[Yy]$ ]]; then
+        GEOMESA_LIB=${GEOMESA_HOME}/lib
+        echo >&2 ""
+        echo "Now set to ${GEOMESA_LIB}"
+    fi
+    echo >&2 ""
+fi
+
+if [ -z "$GEOMESA_CONF_DIR" ]; then
+    GEOMESA_CONF_DIR=${GEOMESA_HOME}/conf
+elif [[ $1 = configure ]]; then
+    message="Warning: GEOMESA_CONF_DIR was already set, probably by a prior configuration."
+    message="${message}\n The current value is ${GEOMESA_CONF_DIR}."
+    echo >&2 ""
+    echo -e >&2 "$message"
+    echo >&2 ""
+    read -p "Do you want to reset this to ${GEOMESA_HOME}/conf? Y\n " -n 1 -r
+    if [[  $REPLY =~ ^[Yy]$ ]]; then
+        GEOMESA_CONF_DIR=${GEOMESA_HOME}/conf
+        echo >&2 ""
+        echo "Now set to ${GEOMESA_CONF_DIR}"
+    fi
+    echo >&2 ""
+fi
+
+GEOMESA_OPTS="${GEOMESA_OPTS} -Dlog4j.configuration=file://${GEOMESA_CONF_DIR}/log4j.properties"
+
+function findJars() {
+    home="$1"
+    CP=""
+    if [ -n "$home" -a -d "$home" ]; then
+        for jar in $(find ${home} -name "*.jar"); do
+            if [[ ("$2" != "true") || (("$jar" != *"test"*) && ("$jar" != *"slf4j"*)) ]]; then
+                if [ "$jar" = "${jar%-sources.jar}" -a "$jar" = "${jar%-test.jar}" ]; then
+                  CP="$CP:$jar"
+                fi
+            fi
+        done
+        if [ -d "$home/native" ]; then
+            if [ -z "$JAVA_LIBRARY_PATH" ]; then
+                JAVA_LIBRARY_PATH="$home/native"
+            else
+                JAVA_LIBRARY_PATH="$home/native:$JAVA_LIBRARY_PATH"
+            fi
+        fi
+    fi
+    if [ "${CP:0:1}" = ":" ]; then
+        CP="${CP:1}"
+    fi
+    echo $CP
+}
+
+# Start constructing GEOMESA_CP (classpath)
+# include geomesa first so that the correct log4j.properties is picked up
+GEOMESA_CP="${GEOMESA_CONF_DIR}:$(findJars $GEOMESA_LIB):$(findJars $GEOMESA_HOME/kafka-lib)"
```

Comment:
@Afterglow375 @elahrvivaz what do you think about making the dir layout be:
```
lib/common
lib/kafka
lib/accumulo
```
like that. so we don't cram the top level dir. For now the current things going into lib would all go into either ```lib/accumulo``` or ```lib/common``` or whatever. But nothing would be in plan ```lib``` except subdirs. Similar to how hadoop does it.